packages/frontend/app/(dashboard)/course/[cid]/queue/[qid]/components/StaffList.tsx

```diff
@@ -8,11 +8,12 @@ import { formatWaitTime } from '@/app/utils/timeFormatUtils'
 
 interface StaffListProps {
   queueId: number
+  mobile?: boolean
 }
```

Comment:
this mobile prop seems unused

Reply:
Nice catch, probably something I was doing with the styling before I abandoned it and forgot to clean up.